src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -1708,6 +1763,14 @@ def pick_strategies(strategy_pool, fuzzer_path, corpus_directory,
     fuzzing_strategies.append(strategy.MUTATOR_PLUGIN_STRATEGY.name)
 
   if (strategy.MUTATOR_PLUGIN_STRATEGY.name not in fuzzing_strategies and
+      strategy_pool.do_strategy(strategy.PEACH_GRAMMAR_MUTATION_STRATEGY) and
```

Comment:
nit: this is getting a bit difficult to read and will get even more so if we add more.

maybe do something like

```python
MUTATOR_STRATAGIES = [
  strategy.PEACH_GRAMMAR_MUTATION_STRATEGY.name,
  strategy.MUTATOR_PLUGIN_STRATEGY.name,
  strategy.MUTATOR_PLUGIN_RADAMSA_STRATEGY.name,
]

def has_existing_mutator_strategy(fuzzing_strategies):
  return any(strategy in fuzzing_strategies for strategy in MUTATOR_STRATEGIES)

...

if (not has_existing_mutator_strategy(fuzzing_strategies) and 
    strategy_pool.do_strategy(..)):

...

Reply:
Done